modules/programs/htop.nix

```
@@ -170,9 +170,19 @@ in {
         ];
       };
 
+      before = if (hasAttr "header_layout" cfg.settings) then {
+        header_layout = cfg.settings.header_layout;
+      } else
+        { };
```

Please use `optionalAttrs` instead:

```suggestion
      before = optionalAttrs (cfg.settings ? header_layout) {
        inherit (cfg.settings) header_layout;
      };
```

Might need to be formatted by the format script.